### PR TITLE
a function to count properties in object

### PR DESCRIPTION
**Pull request by Dove-white**
## Summary
# A function to count the properties in an object and return in number.

## Exercise [Link](https://docs.google.com/document/d/1yi42NaXktb2BQJvuk6cOLflKZ5RELVxXNQ4YPR8oH_E/edit)

### CODE REVIEW

File: count_propertiesInObject.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Here are a few additional requirements for the challenge:
+
+NOTE:
+- You should not use any external libraries or built-in functions that directly give you the count of properties.
+- You should not modify the object or its prototype.
+*/
+
+// -Define an object
+// -Initialize a variable to hold the number zero
```
Comment (SekoViper) [inline count_propertiesInObject.js]: Kindly correct this sentence to read:  Initialize a variable to hold the count

File: count_propertiesInObject.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+const person = {
+    name: 'John',
+    age: 30,
+    occupation: 'Developer'
+};
+
+
+function countObjectProperties(person) {
```
Comment (SekoViper) [inline count_propertiesInObject.js]: The parameter of the function should have a generic name (e.g. obj) instead. What if in the future your object type change (e.g food)?

**Review by SekoViper — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: 
Hello @Dove-white,
Your code runs well without any errors and is very clean. But there are few things to do. Please check the comments I think they are very important. 
Happy coding :computer:

**Comment by Dove-white** [comment]
## Thanks for the corrections

**Pull request merged**